apps/condo/domains/banking/schema/CreateBankAccountRequestService.test.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * Generated by `createservice banking.CreateBankAccountRequestService '--type=mutations'`
+ */
+const faker = require('faker')
+const { makeLoggedInAdminClient, makeClient, expectToThrowGQLError, expectToThrowAccessDeniedErrorToResult } = require('@open-condo/keystone/test.utils')
+const { GQLErrorCode: { BAD_USER_INPUT } } = require('@open-condo/keystone/errors')
+const { expectToThrowAuthenticationErrorToResult, waitFor } = require('@open-condo/keystone/test.utils')
+const { makeClientWithProperty } = require('@condo/domains/property/utils/testSchema')
+const { createBankAccountRequestByTestClient } = require('@condo/domains/banking/utils/testSchema')
+const { INCORRECT_PROPERTY_ID } = require('@condo/domains/banking/constants')
+const { Message } = require('@condo/domains/notification/utils/testSchema')
+const { MESSAGE_SENT_STATUS } = require('@condo/domains/notification/constants/constants')
+
+let admin, client, wrongClient, anonymousClient
+
+describe('CreateBankAccountRequestService', () => {
+    beforeAll(async () => {
+        admin = await makeLoggedInAdminClient()
+        client = await makeClientWithProperty()
+        wrongClient = await makeClientWithProperty()
+        anonymousClient = await makeClient()
+    })
+
+    test('user: can send create bank account request if has canManageBankAccounts access', async () => {
```

Comment:
What about the opposite case when it does not have `canManageBankAccounts` ability?